pkgs/applications/graphics/rawtherapee/default.nix

```diff
@@ -22,6 +22,8 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DPROC_TARGET_NUMBER=2"
+    "-DCACHE_NAME_SUFFIX=\"\""
+    "-DCMAKE_BUILD_TYPE=\"release\""
```

Comment:
`release` is the default build type: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L52

Reply:
Will fix it this evening, thanks!